moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -136,27 +151,29 @@ struct CollisionData
   {
   }
 
-  /// Compute \e active_components_only_ based on \e req_
+  /** \brief Compute \e active_components_only_ based on \e req_ */
```

Comment:
Not sure this is entirely clear, but I won't complain

Reply:
Changed to:

> /** \brief Compute \e active_components_only_ based on the joint group specified in \e req_ */